examples/FunctionalFlavour/Aggregate/UserDescription.php

```diff
@@ -138,15 +140,20 @@ private static function describeConnectWithFriend(EventEngine $eventEngine): voi
     {
         $eventEngine->process(Command::CONNECT_WITH_FRIEND)
             ->withExisting(Aggregate::USER)
+            ->provideContext(SocialPlatformProvider::class)
```

Comment:
Maybe variadic would be useful here. /cc @shochdoerfer 
```
    public function provideContext(string ...$contextProvider): self
    {
        $this->contextProviders[] = $contextProvider;

        return $this;
    }
```

Reply:
The idea is good. I thought about the same when introducing `provideService` but decided against it so that the multiple calls of `provideService` better match with the arguments of the aggregate function. Same for `provideService`.

When doing:

```php
->provideContext(
  ContextProvider1::class,
  ContextProvider2::class,
  ....
)
```

It looks like one context is created from different providers, which is not the case. That's the reason why I would not introduce it.